### DISCUSSION TITLE
Referente ao Módulo Entity Framework, aula testanto os relacionamentos

### DISCUSSION THREAD

**Discussion by marcelocaselli** [balta-io/forum#577] [Tira Dúvidas] [answered]
Bom dia, estou tendo o seguinte erro ao tentar inserir novo usuário no banco:
![erroEntity](https://github.com/balta-io/forum/assets/111652069/8b39e343-16f4-41b5-90e6-241fdc8a7b47)

**Comment by marcelocaselli** [comment]
poderia ajudar ? não consegui resolver

**Comment by andrebaltieri** [comment, marked as answer]
@marcelocaselli boa tarde!
Manda um print do seu `Program.cs` por favor, deu problema na linha 10, mas não deu pra ver na sua imagem

**Comment by marcelocaselli** [comment]
![printProgram](https://github.com/balta-io/forum/assets/111652069/685c7332-2256-454a-a895-d043328f4dcb)

**Comment by marcelocaselli** [comment]
desconsidere o print acima, eu acabei mudando para ir direto fazer a migration
